lib/trailblazer/rails/railtie.rb

```diff
@@ -3,8 +3,37 @@
 
 module Trailblazer
   class Railtie < ::Rails::Railtie
+    def self.load_root(root)
+
+      # Loader has no concept for base dir, must chdir so that it can find files local to the root
+      Dir.chdir(root) do
+        # For STI models, we need to sort all models before concepts, gather files first from loader
+        files = []
+        Loader.new.({debug: false, insert: [ModelFile, before: Loader::AddConceptFiles]}) { |file|
```

Comment:
@apotonick For me this feature would be usefull, too. Even if it is not 100% perfect, the solution from @rosskevin works for me. What if you accept this PR, release a new version and optimize later (or let others optimize)? (Maybe I don't fully understand the problem on loading the models per concept. Is it just a performance hit or are there other traps?)